src/main/java/org/broadinstitute/hellbender/utils/io/IOUtils.java

```diff
@@ -864,4 +906,85 @@ public static final String urlDecode(final String string) {
             throw new UserException("Could not decode sample name", ex);
         }
     }
+
+    /**
+     * Check if a given path represents GenomicsDB URI.
+     *
+     * @param path String containing the path to test
+     * @return true if path represents a GenomicsDB URI, otherwise false
+     */
+    public static boolean isGenomicsDBPath(final String path) {
+        return getGenomicsDBPath(path) != null;
+    }
+
+    /**
+     * Get the GenomicsDB equivalent absolute URL for a given path
+     *
+     * @param gendbPath String representing legal gendb URI
+     * @return absolute gendb URI to the path
+     */
+    public static String getAbsolutePathWithGenDBScheme(final String gendbPath) {
```

Comment:
I'd rather not abbreviate GenomicsDB in this method name. Maybe "getAbsolutePathWithGenomicsDBURIScheme"?

Reply:
Done. See [commit](url
https://github.com/broadinstitute/gatk/commit/6c860bd4559e3b2f3af612fe95d5c0efa4b6a391).